micromanager_gui/_util.py

```diff
@@ -170,3 +173,13 @@ def __init__(
 
     def currentText(self) -> str:
         return self._combo.currentText()
+
+
+def set_wdg_color(color: str, wdg: QWidget):
+    wdg.setStyleSheet(f"color: {color};")
+
+
+def get_dev_prop(group: str, preset: str, mmcore: Optional[CMMCorePlus] = None) -> list:
+    """return a list with (device, property) for the selected group preset"""
+    mmc = mmcore or get_core_singleton()
+    return [(k[0], k[1]) for k in mmc.getConfigData(group, preset)]
```

Comment:
if you do it like this then I think we can eliminate `_check_if_presets_have_same_props`

```suggestion
def get_dev_prop(
    group: str, preset: str, mmcore: Optional[CMMCorePlus] = None
) -> List[Tuple[str, str]]:
    """
    Return a list of all (device, property) tuples used in the config group's presets
    """
    mmc = mmcore or get_core_singleton()
    dev_props = []
    for preset in mmc.getAvailableConfigs(group):
        dev_props.extend([(k[0], k[1]) for k in mmc.getConfigData(group, preset)])
    return dev_props
```